ci/Jenkinsfile.macos

```diff
@@ -1,4 +1,4 @@
-library 'status-react-jenkins@v1.1.7'
+library 'status-react-jenkins@ios-pr-builds'
```

Comment:
You didn't update this one, also there's other Jenkinsfiles in subfolders.

Reply:
Should be correct now